nautobot/extras/tests/test_models.py

```diff
@@ -342,3 +343,4 @@ def test_name(self):
             self.status.name = test
             self.status.clean()
             self.status.save()
+            self.assertEquals(str(self.status), self.status.name)
```

Comment:
While this is a valid assertion, it is not actually testing against the input of `test` to see if the internal logic is in any way manipulating the input value.

Reply:
Oh yeah, great call thanks.